jupyter_book/create.py

```diff
@@ -64,46 +63,56 @@ def update_config(path_to_config, new_config):
         yaml.dump(data, ff)
 
 
-def new_book():
-    """Create a new Jupyter Book."""
-    parser = argparse.ArgumentParser(description="Create a new Jupyter Book")
-    parser.add_argument(
-        "name", help="The name of your Jupyter Book (your book template will be placed in a folder of this name)")
-    parser.add_argument("--out-folder", default='.',
-                        help="The location where your book will be placed")
-    parser.add_argument("--license", default=None,
-                        help="A path to a LICENSE.md file if you have already created one")
-    parser.add_argument("--content-folder", default=None,
-                        help="A path to a folder that holds your book content")
-    parser.add_argument("--toc", default=None,
-                        help="A path to a yaml file that contains a Table of Contents for your Jupyter Book. This will overwrite parts of the book template's default toc.yml configuration")
-    parser.add_argument("--config", default=None,
-                        help="A path to a configuration YAML file that contains configuration for your Jupyter Book. This will overwrite parts of the book template's default _config.yml configuration")
-    parser.add_argument("--custom-css", default=None,
-                        help="A path to a CSS file that defines some custom CSS rules for your book")
-    parser.add_argument("--custom-js", default=None,
-                        help="A path to a JS file that defines some custom CSS rules for your book")
```

Comment:
@choldgraf this seemed like an awkward help statement to me. But I let it be as such because I don't know the inner workings of jupyter-book yet. Why would a JS file define CSS rules?

Reply:
oops, this should be "JS" not "CSS"!
```suggestion
                        help="A path to a JS file that defines some custom JavaScript code for your book")
```